infra/modules/service/scheduled_jobs.tf

```diff
@@ -1,3 +1,13 @@
+locals {
+  scheduled_job_environment_variables = [
+    for name, value in var.scheduled_jobs :
+    {
+      name  = "SCHEDULED_JOB_NAME",
+      value = name
+    }
+  ]
+}
```

Comment:
Could be that I'm not familiar with terraform, but how does this individually give each job its specific environment variable? This reads like it would set the same env vars for every job?

Reply:
It's going to be the same env var for each job, yeah. I just wanna have this env var piped straight into the logs into a `scheduled_job_name` json key

Reply:
Oh yeah I did this wrong, not on my game today 🤦🏼‍♀️ please hold

Reply:
<img width="458" alt="image" src="https://github.com/user-attachments/assets/7bf39fcd-72f4-4a41-a254-f1ad1b528f0d" />